File: Pidev-mobile/src/com/codenameone1/uikit/entities/Evenement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.codenameone1.uikit.entities;

import java.util.Date;

/**
 *
 * @author mondh
 */
public class Evenement {
    private int id,nbrParticipants,nbrPlaces;
    private String nom,type,description;
    private Date DateDebut,DateFin;

    public Evenement(int id, int nbrParticipants, int nbrPlaces, String nom, String type, String description, Date DateDebut, Date DateFin) {
        this.id = id;
        this.nbrParticipants = nbrParticipants;
        this.nbrPlaces = nbrPlaces;
        this.nom = nom;
        this.type = type;
        this.description = description;
        this.DateDebut = DateDebut;
        this.DateFin = DateFin;
    }

    public Evenement() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getNbrParticipants() {
        return nbrParticipants;
    }

    public void setNbrParticipants(int nbrParticipants) {
        this.nbrParticipants = nbrParticipants;
    }

    public int getNbrPlaces() {
        return nbrPlaces;
    }

    public void setNbrPlaces(int nbrPlaces) {
        this.nbrPlaces = nbrPlaces;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Date getDateDebut() {
        return DateDebut;
    }

    public void setDateDebut(Date DateDebut) {
        this.DateDebut = DateDebut;
    }

    public Date getDateFin() {
        return DateFin;
    }

    public void setDateFin(Date DateFin) {
        this.DateFin = DateFin;
    }

    @Override
    public String toString() {
        return "Evenement{" + "id=" + id + ", nbrParticipants=" + nbrParticipants + ", nbrPlaces=" + nbrPlaces + ", nom=" + nom + ", type=" + type + ", description=" + description + ", DateDebut=" + DateDebut + ", DateFin=" + DateFin + '}';
    }
    
    
    
    
}

File: Pidev/src/ProjetBundle/Entity/Evenement.php
<?php

namespace ProjetBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
use Symfony\Component\Serializer\Encoder\JsonEncoder;
use Symfony\Component\HttpFoundation\Response;


/**
 * Evenement
 *
 *
 * @ORM\Table(name="evenement")
 * @ORM\Entity(repositoryClass="ProjetBundle\Repository\EvenementRepository")
 */
class Evenement
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom", type="string", length=255, unique=true)
     */
    private $nomE;



    /**
     * @var string
     *
     * @ORM\Column(name="Type", type="string", length=255)
     */
    private $type;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date_Debut", type="date", unique=true)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date_Fin", type="date", unique=true)
     */
    private $dateFin;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=255, nullable=true, unique=false)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="NbrParticipants", type="integer", nullable=true)
     */
    private $nbrParticipants;

    /**
     * @var int
     *
     * @ORM\Column(name="nbrPlaces", type="integer", nullable=true)
     */
    private $nbrPlaces;



    /**
     * @ORM\OneToMany(targetEntity="ProjetBundle\Entity\Participant",mappedBy="evenement")
     */

    private $participant;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @return int
     */
    public function getNbrPlaces()
    {
        return $this->nbrPlaces;
    }

    /**
     * @param int $nbrPlaces
     */
    public function setNbrPlaces($nbrPlaces)
    {
        $this->nbrPlaces = $nbrPlaces;
    }

    /**
     * Set nomE
     *
     * @param string $nomE
     *
     * @return Evenement
     */
    public function setNomE($nomE)
    {
        $this->nomE = $nomE;

        return $this;
    }

    /**
     * Get nomE
     *
     * @return string
     */
    public function getNomE()
    {
        return $this->nomE;
    }

    /**
     * Set type
     *
     * @param string $type
     *
     * @return Evenement
     */
    public function setType($type)
    {
        $this->type = $type;

        return $this;
    }

    /**
     * Get type
     *
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * Set dateDebut
     *
     * @param \DateTime $dateDebut
     *
     * @return Evenement
     */
    public function setDateDebut($dateDebut)
    {
        $this->dateDebut = $dateDebut;

        return $this;
    }

    /**
     * Get dateDebut
     *
     * @return \DateTime
     */
    public function getDateDebut()
    {
        return $this->dateDebut;
    }

    /**
     * Set dateFin
     *
     * @param \DateTime $dateFin
     *
     * @return Evenement
     */
    public function setDateFin($dateFin)
    {
        $this->dateFin = $dateFin;

        return $this;
    }

    /**
     * Get dateFin
     *
     * @return \DateTime
     */
    public function getDateFin()
    {
        return $this->dateFin;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Evenement
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set nbrParticipants
     *
     * @param integer $nbrParticipants
     *
     * @return Evenement
     */
    public function setNbrParticipants($nbrParticipants)
    {
        $this->nbrParticipants = $nbrParticipants;

        return $this;
    }

    /**
     * Get nbrParticipants
     *
     * @return int
     */
    public function getNbrParticipants()
    {
        return $this->nbrParticipants;
    }
    /**
     * Set participant
     *
     * @param string $participant
     *
     * @return Evenement
     */
    public function setParticipant($participant)
    {
        $this->participant = $participant;

        return $this;
    }

    /**
     * Get participant
     *
     * @return string
     */
    public function getParticipant()
    {
        return $this->participant;
    }
    public function __toString()
    {
        // TODO: Implement __toString() method.
        return $this->nomE;
    }

    /**
     * Add nbrParticipants
     *
     * @param integer $nbrParticipants
     *
     * @return Evenement
     */
    public function AddNbrParticipants()
    {
        $this->nbrParticipants+=1;

        return $this;
    }
    /**
     * Remove nbrParticipants
     *
     * @param integer $nbrParticipants
     *
     * @return Evenement
     */
    public function RemoveNbrParticipants()
    {
        $this->nbrParticipants-=1;

        return $this;
    }

}

